queries.sql

```diff
@@ -5,4 +5,61 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE RIGHT(name, 3) = 'mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE RIGHT(name, 3) != 'mon';
+
+DELETE FROM ANIMALS;
+
+ROLLBACK;
+
+DELETE FROM ANIMALS
+WHERE date_of_birth > '2022-01-01';
+
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
```

Comment:
- [ ] Kindly make these one transaction based on the requirement. Also, please add a `SAVEPOINT` after the statement that deletes all animals born after **_Jan 1st, 2022_**.

### Project Requirement

![image](https://user-images.githubusercontent.com/77106962/228275663-d5d07e2e-3889-4805-92c4-4ed2f946b908.png)

#### Hint: Remember to start the transaction with a `BEGIN` statement. 👍 💻